nixos/modules/services/web-servers/meguca.nix

```diff
@@ -1,65 +1,72 @@
 { config, lib, pkgs, ... }:
 
-with lib;
 let
   cfg = config.services.meguca;
   postgres = config.services.postgresql;
-in
-{
+in with lib; {
   options.services.meguca = {
     enable = mkEnableOption "meguca";
 
-    baseDir = mkOption {
+    dataDir = mkOption {
```

Comment:
This will break configs of people using the `baseDir` option. You'd need to use `mkRenamedOptionModule` to hint at a rename.

Reply:
Like this, right?
```
imports = [
    (mkRenamedOptionModule [ "baseDir" ] [ "dataDir" ])
  ];
```

Reply:
Yup, but you need to use the full paths to the options